config/helpers.php

```diff
@@ -428,6 +428,43 @@ function kirbytext(string $text = null, array $data = []): string
     return App::instance()->kirbytext($text, $data);
 }
 
+/**
+ * Shortcut for `kirbytext()` helper
+ *
+ * @param string $text
+ * @param array $data
+ * @return string
+ */
+function kt(string $text = null, array $data = []): string
+{
+    return kirbytext($text, $data);
+}
+
+/**
+ * Parses KirbyTags and inline Markdown in the
+ * given string.
+ *
+ * @param string $text
+ * @param array $data
+ * @return string
+ */
+function kirbytextinline(string $text = null, array $data = []): string
```

Comment:
I think we should use `camelCase` for this: `kirbytextInline`. Same for all other occurrences.

Reply:
Let's wait to hear from @bastianallgeier if all else would be ok, before making that change.

Reply:
I kept it like the rest. Otherwise we would need to change kirbyText, kirbyTags, etc. as well.